mass_editing/models/ir_model_fields.py

```diff
@@ -20,26 +20,22 @@
 #
 ##############################################################################
```

Comment:
Please change to new OCA header https://github.com/OCA/maintainer-tools/blob/master/template/module/hooks.py#L3